lib/fake_braintree/asset_versions.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.